### PR TITLE
Convert to dark theme

### PR DESCRIPTION
**Pull request by kateloschinina**

### CODE REVIEW

File: .sass-lint.yml [hunk 1]
```diff
@@ -0,0 +1,220 @@
+linters:
```
Comment (ciprianlujeru) [inline .sass-lint.yml]: Any particular reason for `.sass-lint.yml` when we already have `.scss-lint.yml` ?

Comment (kateloschinina) [inline .sass-lint.yml]: I have issue with linter plugin on my Atom, can not figure out what's causing it, but creation of this file fixes it.

Comment (kateloschinina) [inline .sass-lint.yml]: We can put in in git ignore.

Comment (ciprianlujeru) [inline .sass-lint.yml]: Yup, please do. We already have `.scss-lint.yml` and you can just keep the `.sass-lint.yml` locally on your machine

Comment (kateloschinina) [inline .sass-lint.yml]: Super, done!

**Pull request merged**